app/javascript/app/components/about/about-faq/about-faq-data.js

```diff
@@ -192,20 +192,20 @@ export const sectionsData = [
+          'UNFCCC data provide official country-reported inventories; CAIT and PIK datasets are estimates made by non-government research institutes. CAIT and PIK datasets apply a consistent methodology and are comparable across all countries included. The UNFCCC dataset is not comparable across all countries due to the different reporting requirements and methodologies for Annex 1 and non-Annex 1 countries. Non-Annex 1 countries also do not report a complete time series of data for each year. The PIK dataset prioritizes country-reported inventories and fills the data gaps with secondary sources and assumptions. CAIT uses none of the self-reported inventories for individual countries and instead uses datasets with global coverage. Thus, the PIK dataset aligns better with country-reported estimates, while CAIT provides an independent inventory that complements the other two. Among other methodology differences, CAIT uses global warming potential (GWP) values from the IPCC’s Second Assessment Report (AR2) for non-CO2 gases; PIK provides data under both AR2 and AR4 GWP values; UNFCCC data uses either AR2 and AR4, depending on the inventory methodology the party opts to follow. '
       },
       {
         type: 'html',
         title:
           'How significant are uncertainties in the emissions data, particularly in the land-use change and forestry sector? ',
         answer:
-          'According to the Working Group III Contribution to the IPCC Fifth Assessment Report, global CO<sub>2</sub> emissions from fossil fuel combustion are known within 8% uncertainty (90% confidence interval). CO<sub>2</sub> emissions from FOLU (forestry and other land use) have very large uncertainties attached in the order of ± 50%. Uncertainty for global emissions of CH<sub>4</sub>, N<sub>2</sub>O and the F-gases has been estimated as 20%, 60% and 20% respectively. For the PIK dataset, regional deforestation emissions are downscaled to country level using estimates of the deforested area obtained from potential vegetation and calculations for the needed agricultural land. So, levels of uncertainty are quite high for earlier years of data (closer to 1850).  '
+          'According to the Working Group III Contribution to the IPCC Fifth Assessment Report, global CO<sub>2</sub> emissions from fossil fuel combustion are known within 8% uncertainty (90% confidence interval). CO<sub>2</sub> emissions from FOLU (forestry and other land use) have very large uncertainties associated with them in the order of ± 50%. Uncertainty for global emissions of CH<sub>4</sub>, N<sub>2</sub>O and the F-gases has been estimated as 20%, 60% and 20% respectively. For the PIK dataset, regional deforestation emissions are downscaled to the country level using estimates of the deforested area obtained from potential vegetation and calculations for the needed agricultural land. Accordingly, levels of uncertainty are quite high for earlier years of data (closer to 1850).  '
```

Comment:
On point three of `Historical emissions module` FAQ, in the last line, there's a coma after `Accordingly, levels...` that is not present on the corrections document.
![image](https://user-images.githubusercontent.com/6906348/54434327-4c8eb780-472e-11e9-9ae7-a7f7d151fc19.png)

Reply:
Thanks, fixed!